### PR TITLE
Done easy admin

### PR DESCRIPTION
**Pull request by Montekarl**
Initial look is done. Next step is to create relations

### CODE REVIEW

File: config/packages/security.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            # https://symfony.com/doc/current/security.html#firewalls-authentication
+
+            # https://symfony.com/doc/current/security/impersonating_user.html
+            # switch_user: true
+
+    # Easy way to control access for large sections of your site
+    # Note: Only the *first* access control that matches will be used
+    access_control:
+        # - { path: ^/admin, roles: ROLE_ADMIN }
+        # - { path: ^/profile, roles: ROLE_USER }
```
Comment (zmoginas) [inline config/packages/security.yaml]: šiuo metu bet kas gali prisijungti prie sistemos? :thinking:

File: src/Controller/HomeController.php [hunk 2]
```diff
@@ -9,7 +9,7 @@
 class HomeController extends AbstractController
 {
     /**
-     * @Route("/{reactRouting}", name="home", defaults={"reactRouting": null})
+     * @Route("/", name="home", defaults={"": null})
```
Comment (zmoginas) [inline src/Controller/HomeController.php]: Jeigu default reikšmių nėra, tuomet geriau ir nerašyti :sweat_smile: 

```
     * @Route("/", name="home")
```

File: src/Entity/Projectdetails.php [hunk 3]
```diff
@@ -0,0 +1,210 @@
+<?php
+
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: missing declare(strict_types=1);

File: src/Entity/Projectdetails.php [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\ProjectdetailsRepository")
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]:
```
/**
 * @ORM\Entity()
 * @ORM\Table()
 */
```

File: src/Entity/Projectdetails.php [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+ */
+class Projectdetails
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: Prie `$id` siūlau rašyti visur taip:
```
    /**
     * @var int
     * @ORM\Column(type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
```

File: src/Entity/Projectdetails.php [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+    private $id;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $projectfk;
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: kas tas fk?

File: src/Entity/Projectdetails.php [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private $deadline;
+
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private $sprint1;
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: nelabai suprantu kas yra sprint1 :thinking:

File: src/Entity/Projectdetails.php [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $sprint1points;
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: same as above

File: src/Entity/Projectdetails.php [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
+
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private $sprint2;
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: sprint2?

File: src/Entity/Projectdetails.php [hunk 10]
```diff
@@ -0,0 +1,210 @@
...
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $sprint2points;
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: same as above

File: src/Entity/Projectdetails.php [hunk 11]
```diff
@@ -0,0 +1,210 @@
...
+
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private $sprint3;
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: sprint3?

File: src/Entity/Projectdetails.php [hunk 12]
```diff
@@ -0,0 +1,210 @@
...
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $sprint3points;
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: same as above

File: src/Entity/Projectdetails.php [hunk 13]
```diff
@@ -0,0 +1,210 @@
...
+
+    /**
+     * @ORM\Column(type="boolean", nullable=true)
+     */
+    private $status;
+
+    public function getId(): ?int
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: missing annotation

File: src/Entity/Projectdetails.php [hunk 14]
```diff
@@ -0,0 +1,210 @@
...
+    {
+        return $this->id;
+    }
+
+    public function getProjectfk(): ?int
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: missing annotations.
Jeigu gali tai sudėk prie visų metodų anotacijas :smile:

File: src/Entity/Projects.php [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $title;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $teamfk;
```
Comment (zmoginas) [inline src/Entity/Projects.php]: what is teamfk?

File: src/Entity/Projects.php [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+
+    public function getId(): ?int
```
Comment (zmoginas) [inline src/Entity/Projects.php]: same as above about annotations

File: src/Entity/Projects.php [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\ProjectsRepository")
+ */
+class Projects
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
```
Comment (zmoginas) [inline src/Entity/Projects.php]: same as above about `$id`

File: src/Entity/Projectdetails.php [hunk 18]
```diff
@@ -0,0 +1,210 @@
...
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="integer")
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: Vien apibrėžti koks table column bus, neužtenka. Reikia dar apibrėžti ir koks bus property type.
P.S. Apibrėžti reiktų visus `object properties`

Comment (zmoginas) [reply src/Entity/Projectdetails.php]: Jeigu bus neaišku klausk

File: src/Entity/Projects.php [hunk 19]
```diff
@@ -0,0 +1,57 @@
...
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
```
Comment (zmoginas) [inline src/Entity/Projects.php]: same as above, about properties type

File: src/Entity/Projectdetails.php [hunk 20]
```diff
@@ -0,0 +1,210 @@
+<?php
+
+namespace App\Entity;
+
```
Comment (zmoginas) [inline src/Entity/Projectdetails.php]: missing declare(strict_types=1);

File: src/Entity/Projects.php [hunk 21]
```diff
@@ -0,0 +1,57 @@
+<?php
+
```
Comment (zmoginas) [inline src/Entity/Projects.php]: missing declare(strict_types=1);

File: src/Entity/Skill.php [hunk 22]
```diff
@@ -0,0 +1,40 @@
+<?php
+
```
Comment (zmoginas) [inline src/Entity/Skill.php]: missing declare(strict_types=1);
kiekvienas failas turi turėti strict_type.

File: src/Entity/Skill.php [hunk 23]
```diff
@@ -0,0 +1,40 @@
...
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\SkillRepository")
+ */
+class Skill
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
```
Comment (zmoginas) [inline src/Entity/Skill.php]: Same as above about property types, annotations

File: src/Entity/Skills.php [hunk 24]
```diff
@@ -0,0 +1,57 @@
...
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\SkillsRepository")
+ */
+class Skills
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
```
Comment (zmoginas) [inline src/Entity/Skills.php]: Same as above about property types, annotations

File: src/Entity/Skills.php [hunk 25]
```diff
@@ -0,0 +1,57 @@
...
+    private $id;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $userfk;
```
Comment (zmoginas) [inline src/Entity/Skills.php]: userfk?

File: src/Entity/Skills.php [hunk 26]
```diff
@@ -0,0 +1,57 @@
...
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $skillfk;
```
Comment (zmoginas) [inline src/Entity/Skills.php]: skillfk?

File: src/Entity/Teampoints.php [hunk 27]
```diff
@@ -0,0 +1,57 @@
...
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\TeampointsRepository")
+ */
+class Teampoints
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
```
Comment (zmoginas) [inline src/Entity/Teampoints.php]: Same as above about property types, annotations

File: src/Entity/Teampoints.php [hunk 28]
```diff
@@ -0,0 +1,57 @@
...
+    private $id;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $teamfk;
```
Comment (zmoginas) [inline src/Entity/Teampoints.php]: teamfk?

File: src/Entity/Teams.php [hunk 29]
```diff
@@ -0,0 +1,74 @@
...
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\TeamsRepository")
+ */
+class Teams
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
```
Comment (zmoginas) [inline src/Entity/Teams.php]: Same as above about property types, annotations

File: src/Entity/Teams.php [hunk 30]
```diff
@@ -0,0 +1,74 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $githubRepo;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $teampoints;
```
Comment (zmoginas) [inline src/Entity/Teams.php]: should be camelCase

File: src/Entity/Teamtasks.php [hunk 31]
```diff
@@ -0,0 +1,91 @@
...
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\TeamtasksRepository")
+ */
+class Teamtasks
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
```
Comment (zmoginas) [inline src/Entity/Teamtasks.php]: Same as above about property types, annotations

File: src/Entity/Teamtasks.php [hunk 32]
```diff
@@ -0,0 +1,91 @@
...
+    private $id;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $teamfk;
```
Comment (zmoginas) [inline src/Entity/Teamtasks.php]: teamfk?

File: src/Entity/Teamtasks.php [hunk 33]
```diff
@@ -0,0 +1,91 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $task;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $createdby;
```
Comment (zmoginas) [inline src/Entity/Teamtasks.php]: camelCase

File: src/Entity/Users.php [hunk 34]
```diff
@@ -0,0 +1,163 @@
...
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\UsersRepository")
+ */
+class Users
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
```
Comment (zmoginas) [inline src/Entity/Users.php]: Same as above about property types, annotations

File: src/Entity/Users.php [hunk 35]
```diff
@@ -0,0 +1,163 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $lastname;
```
Comment (zmoginas) [inline src/Entity/Users.php]: camelCase

File: src/Entity/Users.php [hunk 36]
```diff
@@ -0,0 +1,163 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $githubUsername;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $teamfk;
```
Comment (zmoginas) [inline src/Entity/Users.php]: teamfk?

File: src/Entity/Users.php [hunk 37]
```diff
@@ -0,0 +1,163 @@
...
+        return $this->teamfk;
+    }
+
+    public function setTeamfk(?int $teamfk): self
+    {
+        $this->teamfk = $teamfk;
+
+        return $this;
+    }
+    public function __toString()
```
Comment (zmoginas) [inline src/Entity/Users.php]: space between methods

File: src/Repository/ProjectdetailsRepository.php [hunk 38]
```diff
@@ -0,0 +1,50 @@
...
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+
+/**
+ * @method Projectdetails|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Projectdetails|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Projectdetails[]    findAll()
+ * @method Projectdetails[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class ProjectdetailsRepository extends ServiceEntityRepository
```
Comment (zmoginas) [inline src/Repository/ProjectdetailsRepository.php]: Jeigu nenaudojame šios repositorijos metodų, tuomet mes galime ją trinti lauk :smile: Sukursime ją naujai kuomet prireiks :sweat_smile:

File: src/Repository/ProjectsRepository.php [hunk 39]
```diff
@@ -0,0 +1,50 @@
...
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+
+/**
+ * @method Projects|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Projects|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Projects[]    findAll()
+ * @method Projects[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class ProjectsRepository extends ServiceEntityRepository
```
Comment (zmoginas) [inline src/Repository/ProjectsRepository.php]: same as above :smile:

File: src/Repository/SkillRepository.php [hunk 40]
```diff
@@ -0,0 +1,50 @@
...
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+
+/**
+ * @method Skill|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Skill|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Skill[]    findAll()
+ * @method Skill[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class SkillRepository extends ServiceEntityRepository
```
Comment (zmoginas) [inline src/Repository/SkillRepository.php]: same as above :smile:

File: src/Repository/SkillsRepository.php [hunk 41]
```diff
@@ -0,0 +1,50 @@
...
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+
+/**
+ * @method Skills|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Skills|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Skills[]    findAll()
+ * @method Skills[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class SkillsRepository extends ServiceEntityRepository
```
Comment (zmoginas) [inline src/Repository/SkillsRepository.php]: same as above :smile:

File: src/Repository/TeampointsRepository.php [hunk 42]
```diff
@@ -0,0 +1,50 @@
...
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+
+/**
+ * @method Teampoints|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Teampoints|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Teampoints[]    findAll()
+ * @method Teampoints[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class TeampointsRepository extends ServiceEntityRepository
```
Comment (zmoginas) [inline src/Repository/TeampointsRepository.php]: same as above :smile:

File: src/Repository/TeamsRepository.php [hunk 43]
```diff
@@ -0,0 +1,50 @@
...
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+
+/**
+ * @method Teams|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Teams|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Teams[]    findAll()
+ * @method Teams[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class TeamsRepository extends ServiceEntityRepository
```
Comment (zmoginas) [inline src/Repository/TeamsRepository.php]: same as above :smile:

File: src/Repository/TeamtasksRepository.php [hunk 44]
```diff
@@ -0,0 +1,50 @@
...
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+
+/**
+ * @method Teamtasks|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Teamtasks|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Teamtasks[]    findAll()
+ * @method Teamtasks[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class TeamtasksRepository extends ServiceEntityRepository
```
Comment (zmoginas) [inline src/Repository/TeamtasksRepository.php]: same as above :smile:

File: src/Repository/UsersRepository.php [hunk 45]
```diff
@@ -0,0 +1,50 @@
...
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+
+/**
+ * @method Users|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Users|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Users[]    findAll()
+ * @method Users[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class UsersRepository extends ServiceEntityRepository
```
Comment (zmoginas) [inline src/Repository/UsersRepository.php]: same as above :smile:

File: src/Entity/Projectdetails.php (hunk 6 above)
Comment (Montekarl) [reply src/Entity/Projectdetails.php]: foreign key. Tikiuosi naudoti sita relationshipams tarp lenteliu

File: src/Entity/Projectdetails.php (hunk 7 above)
Comment (Montekarl) [reply src/Entity/Projectdetails.php]: sprintams zymeti, project managementui.

**Comment by EvaldasBurlingis** [comment]
Karolis added new easy admin pull request

**Pull request closed without merging**